src/css/index.scss

```diff
@@ -62,103 +72,73 @@ h3 {
 
 /*******************this is the sidebar*/
 
-
 .mapPlaces {
     color: var(--text-color);
-    grid-gap: 10px;
-    display: grid;
-    margin-left: 30px;
-    grid-template-columns: 1fr;
-    grid-column-start: 2;
-    grid-row-start: 1;
-    grid-row-end: -1;
+    grid-area: places;
     overflow: hidden;
     overflow-y: scroll;
     max-height: 850px;
-    
-    position: relative !important;
-    & div {
-        width: auto !important;
-        height: auto !important;
-    }
-    & li {
+    li {
         list-style: none;
     }
-    & .listings {
+    .listings {
         border-bottom: var(--border-color) 1px solid;
         display: grid;
         grid-template-columns: 1fr 149px;
         grid-column: auto;
         padding-bottom: 5px;
-        // box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
         align-items: center;
-
-        & .listing-description {
+        .listing-description {
             display: grid;
             grid-template-columns: minmax(100px, 300px);
             grid-template-rows: 1fr 2fr;
         }
-        & .listing-icon-div {
+        .listing-icon-div {
             display: grid;
-
             grid-column-start: 2;
             grid-row-start: 1;
             min-height: 135px;
             min-width: 135px;
             align-items: center;
-            justify-items: center;
+            justify-content: center;
         }
-
-        & .listings-icon {
-
+        .listings-icon {
             width: 135px;
             height: 135px;
-
-
         }
-        & .listing-name {
-            display: grid;
+        .listing-name {
             color: #323232;
             font-weight: 400;
-
         }
-        & .listings-info {
+        .listings-info {
             color: #336282;
-            display: grid;
-
         }
-        & .listing-open {
+        .listing-open {
             font-weight: bold;
-            display: grid;
-            }
+            color: var(--header-color);
+        }
     }
 }
 
 
 
 /****description bottom of the map */
 .mapDescription {
-    display: grid;
-    grid-row-start: 3;
-    height: auto;
-    padding:10px;
+    grid-area: desc;
+    padding: .8em;
 }
 
 /* This is the Search area */
 .searchContainer {
-    display: grid;
-    grid-column: span 2;
-    grid-row: span 1;
-    grid-column-start: 1;
+    grid-area: search;
+    padding: .8em;
+    align-items: flex-end;
```

Comment:
My only critique is that I didn't like the stretched search bar post Milligram's removal. Maybe add a max-width: to a digit you'd prefer?